cypress/components/link/link.cy.tsx

```diff
@@ -72,6 +73,18 @@ context("Test for Link component", () => {
       tooltipPreview().should("have.text", testCypress);
     });
 
```

Comment:
**question**: wondering if these cypress tests would be necessary here, given we are testing the same styling with a jest test?

Reply:
Good question, technically we could probably get away with just having unit tests, but I thought it was worth just doubling up as we've done the same for other features in `Link`

Reply:
Don't really mind if you leave both unit and CI tests in tbh. However I do think a test story so chromatic can capture it would be best